elasticdl/python/worker/worker.py

```diff
@@ -345,7 +347,7 @@ def get_model(self):
         self._model_version = max(self._model_versions_from_ps)
         self._timing.end_record_time("get_model")
 
-    def pull_embedding_vector(self, layer_name, embedding_ids):
+    def pull_embedding_table(self, layer_name, embedding_ids):
         """Pulls and returns embedding vectors ordered by the embedding ids."""
```

Comment:
Why do you rename `pull_embedding_table`? The `pull_embedding_vector` is more suitable for the docstring.

Reply:
It's actually pulling some embedding vectors from an embedding table. So maybe `pull_embedding_vectors` is more accurate.
Rename it.